shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -184,8 +184,9 @@
     if (gpu_thread_merger->IsMerged()) {
       gpu_thread_merger->ExtendLeaseTo(kDefaultMergedLeaseDuration);
     } else {
+      // Wait until |EndFrame| to perform thread merging.
```

Comment:
`to perform thread merging` -> `to merge the threads`.

Reply:
done